public/app/features/dashboard/state/DashboardMigrator.test.ts

```diff
@@ -66,6 +66,7 @@ describe('DashboardModel', () => {
         panels: [
           {
             type: 'graph',
+            // @ts-expect-error
```

Comment:
I'm not sure what to do with this file. Because this tests previous versions of the dashboard schema, they're basically never going to match our current (work in progress) generated schema type....